### ISSUE TITLE
Error: installation not correct

### ISSUE THREAD

**Issue by vitt2022** [grubaughlab/DENV_pipeline#15]
I have followed all the installation instructions, i.e. created the conda environment, activated it and ran `pip install .`
When I tried to run `denv_pipeline -h`, it gives me this error:

`Error: installation not correct. Ensure that environment is activated and you have run 'pip install .'`

How do I fix this?

**Comment by ViralVerity** [comment]
so that error comes up when it can't find snakemake or biopython - in other words it won't run successfully.

This feels to me like it might be an issue with where you're installing things? Try typing ```which biopython``` and ```which snakemake``` to see if they're installed in the place you're expecting them to be (ie in some kind of directory like $USER/conda/envs/analysis_env/bin/snakemake)

If they are where they should be then this could well be a bug in the pipeline and I can take a look!

**Comment by vitt2022** [comment]
Hi Verity!

It seems like both `biopython` and `snakemake` are already installed in the conda environment (pictured below)
![image](https://github.com/grubaughlab/DENV_pipeline/assets/107045565/6ccc794e-eff7-4d98-a9a8-e204f04ea4b3)

**Comment by ViralVerity** [comment]
hmm odd - and when you run the pipeline in there it comes up with that error?

This is a little tricky to diagnose because I am not having this issue on our end!

**Comment by tuyenhn** [comment]
Hi Verity, so I've been supporting Vi with this issue for the past few days, and I would like to share what happened and how did we fix it, in case someone else running into the same issue.

The problem wasn't relating to the package at all but it was my mistake during installation. I forgot to activate the conda environment before running `pip install .`, so python installed the package into the system environment instead. Every time I ran `denv_pipeline`, it was picking up the executable from the system environment, not conda's, which obviously was lacking prerequisite packages. I uninstalled the package from the system environment and it works normally now!
